cmd/main.go

```diff
@@ -641,6 +655,10 @@ func main() {
 		}
 	}()
 
+	if Options.EnableKubeAPI {
+		importLocalCluster(ctrlMgr, log)
```

Comment:
Is there no feature flag for this import, specifically?

Reply:
+1 we probably need to have something that can disable this feature if needed

Reply:
adding a flag to allow feature to be disabled.